utils_pandas.py

```diff
@@ -101,8 +101,8 @@ def human_format(num: float, pos: int) -> str:
         magnitude += 1
         num /= 1000.0
     # add more suffixes if you need them
-    suffix = ['', 'K', 'M', 'G', 'T', 'P'][magnitude]
-    return f'{num:.1f}{suffix}'
+    suffix = ['', 'k', 'M', 'G', 'T', 'P'][magnitude]
```

Comment:
is K intentionally made lower case here?

Reply:
Yes, I think it should be, but might be that I am wrong and that it is more Austrian style.

Reply:
Yes, you are quite correct! Good catch and thanks for the fix! :clap: